File: Java/Car.java
package Java;

class Car {
    Integer id;
    String license;
    Account driver;
    int passenger;
    
    public Car(String license, Account driver){//parametros obligatorios
        this.license = license;
        this.driver = driver;
    }
    void printDataCar(){
        System.out.println("License: " + license + " Name Driver: " + driver.name);
    }   
}
    

File: PHP/Car.php
<?php
require_once("Account.php");
class Car{
    public $id;
    public $license;
    public $driver;
    public $passenger;
    
    public function __construct($license, Account $driver){
        $this->$title = $title;
        $this->$driver = $driver;
    }

    public function printDataCar(){
        print 'License: '.$this->license;
        print 'License: '.$this->driver->name;
    }
}
?>